packages/h1_parser/h1_parser.0.0.2/opam

```diff
@@ -21,7 +21,7 @@ depends: [
   "odoc" {with-doc}
 ]
 conflicts: [
-  "cohttp-lwt-unix" {with-test & >= "5.0.0"}
+  "cohttp" {with-test & >= "5.0.0"}
```

Comment:
`with-test` is not available in `conflicts` sadly. See https://github.com/ocaml/opam/issues/4048